concent_api/core/validation.py

```diff
@@ -119,35 +154,57 @@ def validate_golem_message_signed_with_key(
             'There was an exception when validating if golem_message {} is signed with public key {}'.format(
                 golem_message.TYPE,
                 public_key
-            )
+            ),
+            error_code=ErrorCode.MESSAGE_SIGNATURE_WRONG,
```

Comment:
Same as above. This should be outside of the function so that you can report which specific piece of data has this problem.

Reply:
Same here, lets add it to same separate issue as mentioned above.

Reply:
OK. Please create an issue.